requirements.txt

```diff
@@ -0,0 +1 @@
+cssselect
```

Comment:
you don't need this, you can use xpath

Reply:
I found it cleaner to use this lib (which was initially included in ```lxml```) instead of writing some long obscure xpath expression like: ```a[@class and contains(concat(' ', normalize-space(@class), ' '), ' view_in_browser_url ')]```
But I understand if it's better not to use it. Your opinion ?

Reply:
If you want to select by class, odoo contains one helper: `//*[hasclass("view_in_browser_url")]`. It should be enough.